src/main/java/pl/coderslab/charity/configuration/SecurityConfig.java

```diff
@@ -0,0 +1,37 @@
+package pl.coderslab.charity.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
```

Comment:
Ogólnie wydaje mi się, że to nie pójdzie bo nie masz importu Spring Security Startera w pom.xml

Reply:
zaktualizowałem pom.xml